### PR TITLE
Add reveal in explorer

### LINKED ISSUE #11: Reveal in explorer
I would love to have a "reveal in explorer" option below "reveal in folder".

### PR DESCRIPTION
**Pull request by mariovalney**
Resolves #11.

_Sorry if I change too much in https://github.com/Molunerfinn/VSCode-RevealFileInFolder/commit/45b0e91e0720e3205a3b41f0524c7bee6202cbab: I was following [this deprecation notice](https://www.npmjs.com/package/vscode)._

### CODE REVIEW

**Comment by mariovalney** [comment]
Hey @Molunerfinn !
I would love a review. Thanks again.

File: src/extension.ts [hunk 1]
```diff
@@ -14,15 +14,25 @@ export function activate (context: vscode.ExtensionContext) {
...
   })
 
-  context.subscriptions.push(disposable)
+  context.subscriptions.push(revealFileInFolderCommand)
+
+  const revealFileInExplorerCommand = vscode.commands.registerCommand('extension.revealFileInExplorer', () => {
+    // The code you place here will be executed every time your command is executed
+
+    // Display a message box to the user
+    // vscode.window.showInformationMessage('Hello World!')
```
Comment (Molunerfinn) [inline src/extension.ts:31]: remove this line？

**Review by Molunerfinn — APPROVED** [review]

**Comment by Molunerfinn** [comment]
Sorry for the delay, thanks for your contribution!

**Comment by mariovalney** [comment]
> Sorry for the delay, thanks for your contribution!

No problem! Thanks for your extension. Love it.

**Pull request merged**